src/request/builder.rs

```diff
@@ -400,7 +400,7 @@ impl<B: Body> RequestBuilder<B> {
         header_insert(&mut prepped.base_settings.headers, CONNECTION, "close")?;
         prepped.set_compression()?;
         match prepped.body.kind()? {
-            BodyKind::Empty => (),
```

Comment:
Should we set the Transfer-Encoding header to chunked?

Reply:
Definitely! Again, forgot this while porting. Thank god for code review...

Reply:
Fixed.